codemirror/codemirror.d.ts

```diff
@@ -616,6 +618,7 @@ declare module CodeMirror {
     }
 
     interface Position {
+        new (line: number, ch: number): Position;

```

Comment:
If you do that, can you please add this too?
`Pos` can be called like a function too (and in fact it is in the CodeMirror's source)

```typescript
    (line: number, ch: number): Position;
```